manifests/c/CodecGuide/K-LiteCodecPack/Full/17.3.0/CodecGuide.K-LiteCodecPack.Full.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CodecGuide.K-LiteCodecPack.Full
+PackageVersion: 17.3.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://files3.codecguide.com/K-Lite_Codec_Pack_1730_Full.exe
+  InstallerSha256: 7DE6EAD659BA92206DA418DA58087C7B097E4606F42FCD95DE0F1EC4A959149F
+  AppsAndFeaturesEntries:
+  - DisplayName: K-Lite Codec Pack 17.2.5 Full

```

```suggestion
  - DisplayName: K-Lite Codec Pack 17.3.0 Full
```
This must match Control Panel.